.travis.yml

```diff
@@ -21,6 +21,10 @@ cache:
 before_install:
   - npm install -g npm@5.5.1
   - sudo apt-get install libgconf2-4
+  - export SOURCE_BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
+  - export IS_BETA=$([[ $TRAVIS_TAG =~ .*beta.* ]] && echo true || echo false)
+  - export IS_RELEASE=$([[ $SOURCE_BRANCH =~ ^.*release.*$|^master$ ]] && echo true || echo false)
+  - export IS_FEATURE=$([[ $SOURCE_BRANCH =~ ^JSUI-[0-9]+$ ]] && echo true || echo false)
```

Comment:
These variables are introduced to improve the readability of the conditions.

`IS_RELEASE` is evaluated by the `^.*release.*$|^master$` regular expression instead of the `! $TRAVIS_TAG =~ .*beta.*` regular expression since these conditions are also going to be evaluated on `JSUI-*` branches, from now on.